pkg/scheduler/scheduler.go

```diff
@@ -675,7 +675,7 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 		} else {
 			// Calculating nodeResourceString can be heavy. Avoid it if klog verbosity is below 2.
 			if klog.V(2).Enabled() {
-				klog.Infof("pod %v/%v is bound successfully on node %q, %d nodes evaluated, %d nodes were found feasible.", assumedPod.Namespace, assumedPod.Name, scheduleResult.SuggestedHost, scheduleResult.EvaluatedNodes, scheduleResult.FeasibleNodes)
+				klog.InfoS("Successfully bound pod to node", "pod", klog.KObj(pod), "host", scheduleResult.SuggestedHost, "evaluatedNodes", scheduleResult.EvaluatedNodes, "feasibleNodes", scheduleResult.FeasibleNodes)
```

Comment:
```suggestion
				klog.InfoS("Successfully bound pod to node", "pod", klog.KObj(pod), "suggestedNode", scheduleResult.SuggestedHost, "evaluatedNodes", scheduleResult.EvaluatedNodes, "feasibleNodes", scheduleResult.FeasibleNodes)
```

Reply:
I think we should use "node" as this will be consistent with other logs referencing this node. Using different name would be useful to distinguish between more then one node, which is not a case here.

Reply:
sg